arsenal/data/cheats/Active_directory/responder.md

```diff
@@ -44,6 +44,21 @@ sed -i 's/SMB = Off/SMB = On/g' /opt/tools/Responder/Responder.conf && cat /opt/
 sed -i 's/SMB = On/SMB = Off/g' /opt/tools/Responder/Responder.conf && cat /opt/tools/Responder/Responder.conf | grep --color=never 'SMB ='
 ```
 
+## responder challenge set
+#plateform/linux #target/local #cat/UTILS
+Set the NTLM challenge for cracking
+
+= challenge: 1122334455667788
+```
+sed -i 's/Challenge =.*$/Challenge = <challenge>/g' /opt/tools/Responder/Responder.conf && cat /opt/tools/Responder/Responder.conf | grep --color=never 'Challenge ='
+```
+
+## responder challenge reset
+#plateform/linux #target/local #cat/UTILS
+```
+sed -i 's/Challenge =.*$/Challenge = Random/g' /opt/tools/Responder/Responder.conf && cat /opt/tools/Responder/Responder.conf | grep --color=never 'Challenge ='
```

Comment:
Hi,
I think it would be better to reset the challenge to the original value: "1122334455667788" rather than "Random"

Reply:
Random is the default on a fresh install, so that's why it's what's used in the reset action